server/models/draftReferral.ts

```diff
@@ -14,6 +15,7 @@ export interface ReferralFields {
   furtherInformation: string
   relevantSentenceId: number
   desiredOutcomesIds: string[]
+  cohortDesiredOutcomes: CohortDesiredOutcomes[]
```

Comment:
If this will eventually supersede `desiredOutcomesIds`, should we drop the `cohort` from this?